interactive_templates/schema/v2.py

```diff
@@ -20,7 +29,7 @@ class Analysis:
     filter_population: str
     repo: str
     time_scale: str
-    time_value: int = field(converter=int)
+    time_value: Optional[int] = field(converter=int_or_none)
```

Comment:
I don't think this change is being tested right now, but could be checked in the functional test below.

Reply:
I've changed the default to be None so it's tested in the default test